README.md

```diff
@@ -9,41 +9,50 @@ AIxplora is your new personal assistant, an open-source project that opens up un
 It leverages AI and LLMs to understand all types of documents, unrestricted by their length or format.
 
 **Imagine being able to query PDF files, MP3 audio, videos, and other types of documents with equal ease and proficiency. 
-Yes, that's the limitless world AIxplora is inviting you into!***
+Yes, that's the limitless world AIxplora is inviting you into!**
 
 
-## How Can You Contribute?
-There are several ways you can contribute as I just released a PoC of the project:
-
-- Code: Write, refactor, optimize - every line of code matters!
-
-- Documentation: Help us make our project more understandable and user-friendly.
-
-- Testing: Every bug found is a step towards perfection.
-
-- Suggest Features: We believe in the power of ideas, no matter where they come from.
-
-- Spread the Word: Share our project within your networks. The more people know about AIxplora, the better it can become!
+## Contributing
+We welcome contributors of all backgrounds and interests, testing, accessibility, community building, and writing code. If you would like to contribute code, feel free to check out our  [contributing guide](https://github.com/grumpyp/aixplora/blob/main/CONTRIBUTING.md).
 
 ## Demo and introduction videos
-[![Summary function showcase](https://img.youtube.com/vi/8x9HhWjjNtY/hqdefault.jpg)](https://youtu.be/8x9HhWjjNtY)
-https://youtu.be/2lNNKLM0o7U
-https://youtu.be/eKLmhJobVvc
+- [Summarize documents](https://youtu.be/8x9HhWjjNtY)
+- [Audio query](https://youtu.be/2lNNKLM0o7U)
+- [Vector databases and LLMs query](https://youtu.be/eKLmhJobVvc)
 
```

Comment:
theres a new video here:
https://youtu.be/JW2mX0aJl2U

would you add it please?